eos/form-factors/Makefile.am

```diff
@@ -26,6 +26,7 @@ libeosformfactors_la_SOURCES = \
 	form-factors.cc form-factors.hh form-factors-fwd.hh \
 	hqet-b-to-c.hh hqet-b-to-c.cc \
 	k-lcdas.cc k-lcdas.hh \
+	k-star-lcdas.cc k-star-lcdas.hh \
```

Comment:
In order to avoid a clash of the file names (and then also class names) I had to use names for the files that do not conform with the EOS standards. Are the names as I chose them (also the class names (Anti)KStarLCDAs instead of (Anti)KstarLCDAs) fine or should I change that?

Reply:
Leave it as is. We sort it out in a separate PR replacing the old code / removing the old code.